lhotse/cut.py

```diff
@@ -195,12 +183,10 @@ def pad(self, desired_duration: Seconds) -> AnyCut:
         :param desired_duration: The cut's minimal duration after padding.
         :return: a padded MixedCut if desired_duration is greater than this cut's duration, otherwise self.
         """
-        if self.features is None:
-            # TODO: fill zeros for raw audio
-            return self
         if desired_duration <= self.duration:
             return self
         padding_duration = desired_duration - self.duration
+        assert self.features is not None
```

Comment:
I think this is not needed - PaddingCut implements `load_audio()` that returns a vector of zeros. Audio padding should work just fine (unless I'm missing something?)

Reply:
Although `PaddingCut` implemented `load_audio()`, its properties {`num_frames`, `frame_shift`} are only for features. I think maybe we have to modify `PaddingCut` to support raw audio.